### DISCUSSION TITLE
spf record, dkim

### DISCUSSION THREAD

**Discussion by JluisMC** [postalserver/postal#2182] [Help with using Postal]
Good afternoon POSTAL community, I require your collaboration, I installed POSTAL and the spf and dkim records have been created on my domain server but it does not recognize them, attached image. Has it happened to someone? and how is it solved?
Note: It was also configured in another vps and it does not recognize it either.
![registros](https://user-images.githubusercontent.com/38479423/195720935-efd3b290-7c72-4bfa-87f3-b368c8160c1d.png)

**Comment by uhlhosting** [comment]
Have you queried with dig or some websites for dns propagation if the records are valid? I see you use some example.com you should be using your own domain.

**Reply by JluisMC** [reply]
That's right, I do the review at https://www.whatsmydns.net, 72 hours have passed and it still doesn't recognize it.
Note: instead of example.com is the domain name.

**Reply by uhlhosting** [reply]
Any logs on your postal console?
You should check also issues here maybe someone had same issues.

**Reply by JluisMC** [reply]
Any log on your postal console?
Sorry I did not understand your question, if I was reading that someone had a similar problem but it remained without a solution.

**Comment by willpower232** [comment]
It looks like you haven't finished setting up your Postal server. The config file contains example.com as an example of what you need to replace with one of your own domains.

If you're able to finish that then it will be easier to see where you're going wrong.
